apps/console/src/auth.html

```diff
@@ -223,6 +225,20 @@
                         return serverOrigin + getTenantPath(tenantDomain);
                     }
 
+                    function getADUApiPath(path, tenantDomain) {
```

Comment:
Let's add a doc comment explaining the function, especially what ADU refers to :)